pkg/profiler/profiler.go

```diff
@@ -765,10 +765,10 @@ func (p *CgroupProfiler) writeProfile(ctx context.Context, prof *profile.Profile
 		i              = 0
 	)
 	for key, value := range p.Labels() {
-		labelOldFormat[i] = &profilestorepb.Label{
+		labelOldFormat = append(labelOldFormat, &profilestorepb.Label{
```

Comment:
Let me know if this is the way to Go 😄

Reply:
:sweat_smile: Oppsie. 
I see my mistake now https://go.dev/play/p/YmTg0VvaJY1

Thanks for taking care of it.